src/layouts/components/Header.tsx

```diff
@@ -237,3 +325,143 @@ function HeaderLink(props: {
+            )}
+            text={item.text}
+            pathname={item.pathname}
+            testId={`${viewPort}.HeaderLink.${item.text.replace(" ", "")}`}
+          />
+        );
+      })}
+    </div>
+  );
+}
+
+function MoreDropdown(): JSX.Element {
+  const [isItemClicked, setIsItemClicked] = useState(false);
+  const router = useRouter();
+  useEffect(() => {
+    setIsItemClicked(
+      dropDownLinks.some((ddl) =>
+        router.pathname.includes(ddl.rootPathName.toLowerCase())
+      )
+    );
+  }, [router.pathname]);
+
+  return (
+    <Menu as="div" className="relative">
+      {({ open, close }) => (
+        <>
+          <Menu.Button
+            className={classNames(
+              "flex flex-row items-center mx-4 dark:hover:text-dark-50 cursor-pointer text-lg hover:text-primary-500",
+              {
+                "dark:text-dark-50 text-primary-500": isItemClicked,
+              }
+            )}
+          >
+            More
+            {open ? (
+              <IoChevronUp className="ml-2" size={20} />
+            ) : (
+              <IoChevronDown className="ml-2" size={20} />
+            )}
+          </Menu.Button>
+          <Transition
+            enter="transition duration-100 ease-out"
+            enterFrom="transform scale-95 opacity-0"
+            enterTo="transform scale-100 opacity-100"
+            leave="transition duration-75 ease-out"
+            leaveFrom="transform scale-100 opacity-100"
+            leaveTo="transform scale-95 opacity-0"
+          >
+            <Menu.Items className="absolute m-4 min-w-max flex flex-col divide-y bg-white border rounded-lg border-gray-200 dark:border-gray-700 dark:bg-gray-700">
+              {dropDownLinks.map((item, index) => {
+                return (
+                  <Menu.Item key={index}>
+                    <Link href={{ pathname: item.link }} passHref>
+                      <a
+                        onClick={close}
+                        href={item.link}
+                        className={classNames(
+                          "px-6 py-3.5 cursor-pointer text-sm border-gray-200 hover:text-primary-500 dark:hover:text-dark-50",
+                          {
+                            "dark:text-dark-50 text-primary-500":
+                              router.pathname.includes(item.rootPathName),
+                          }
+                        )}
+                      >
+                        {item.name}
+                      </a>
+                    </Link>
+                  </Menu.Item>
+                );
+              })}
+            </Menu.Items>
+          </Transition>
+        </>
+      )}
+    </Menu>
+  );
+}
+
+const dropDownLinks = [
+  // {
+  //   name: "Consortium",
+  //   link: "/consortium/asset_breakdown",
+  //   rootPathName: "consortium",
+  // },
+  {
+    name: "On-Chain Governance",
+    link: "/on-chain-governance",
+    rootPathName: "on-chain-governance",
+  },
+];
+
+const menuItemLinks = [
```

Comment:
can we rename to `drawerMenuItemLinks`? to indicate that this is only for non-desktop

Reply:
renamed